includes/content-distribution/class-outgoing-post.php

```diff
@@ -202,15 +202,20 @@ public function get_payload_hash( $payload = null ) {
 	 * @return array|WP_Error The post payload or WP_Error if the post is invalid.
 	 */
 	public function get_payload( $status_on_create = 'draft' ) {
+		$post_author = $this->post->post_author ? Outgoing_Author::get_wp_user_for_distribution( $this->post->post_author ) : [];
+		$multiple_authors = apply_filters( 'newspack_network_multiple_authors_for_post', [], $this->post );
+
 		return [
 			'site_url'         => get_bloginfo( 'url' ),
 			'post_id'          => $this->post->ID,
 			'post_url'         => get_permalink( $this->post->ID ),
 			'network_post_id'  => $this->get_network_post_id(),
 			'sites'            => $this->get_distribution(),
 			'status_on_create' => $status_on_create,
+			'multiple_authors' => is_wp_error( $multiple_authors ) ? [] : $multiple_authors,
```

Comment:
Partial payload filters through the contents of `post_data`. This should move there in order to work as designed.

As it is now, at the root-level, it'll be always part of the payload. It works, but not how we expect it to.

Reply:
👍 Moved it to post_data